spec/services/fee_reform/search_offences_spec.rb

```diff
@@ -29,7 +35,7 @@
   end
 
   context 'when search_offence filter is provided' do
-    let(:filters) { { search_offence: 'pattern' } }
+    let(:filters) { { fee_scheme: 'AGFS 10', search_offence: 'pattern' } }
```

Comment:
🤔 is it worth having a separate test for filtering the same offence but in a scheme 10 and scheme 11 filtering context?